.github/workflows/ci.yml

```diff
@@ -2,56 +2,63 @@ name: CI
 
```

Comment:
why not creating a separate MR to update the pipeline and versions? , this would make this MR a little smaller

Reply:
Separate PRs didn't work out the last time since it takes more time and effort ... I thought it would be easier this time to one-shot it :)

Reply:
We can still split it all up if you'd like in the end.

Reply:
thats fine. let s just go with it for now